crosschain-resolver/contracts/L1Resolver.sol

```diff
@@ -163,6 +164,21 @@ contract L1Resolver is EVMFetchTarget, ITargetResolver, IMetadataResolver, IExte
         }
     }
 
+    /** 
+     * @dev Resolve and throws an EIP 3559 compliant error
+     * @param name DNS encoded ENS name to query
+     * @param data The actual calldata
+     * @return result result of the call
+     */
+    function resolveDeferral(bytes calldata name, bytes calldata data) external view returns (bytes memory result) {
```

Comment:
5559 doesn't specify the required format for ENS setter methods; we can dictate that ourselves, though deviating from the current standard will require changing the manager and any other apps that support it.

Why do you need the plaintext name?

Reply:
Sorry, I accidentally deleted your comment instead of my own.

I understand it's DNS encoded - it's still plaintext (Eg, not hashed). If it's needed, we should define a new `setAddr` with the signature you suggest.